positionally.js

```diff
@@ -4,7 +4,8 @@ let positionally = (code, inputs = [], flags = '', output = x => process.stdout.
     let maxlength = Math.max(...code.map(x => x.length));
     code = code.map(x => x.padEnd(maxlength, ' ').split``);
 
-    inputs = inputs.flatMap(x => typeof x == 'string' ? x.split``.map(x => x.charCodeAt()) : x);
+    inputs = inputs.flatMap(x => typeof x == 'string' ? x.split``.map(x => x.charCodeAt()).concat(0) : x);
+    inputs.push(0)
```

Comment:
I still don't think there should be a 0 at the end of all inputs. Besides, how can this be distinguished from a zero-length string?